genet/utils/google_directions.py

```diff
@@ -1,21 +1,23 @@
 import ast
 import itertools
+import json
 import logging
-import polyline
 import os
 import time
-import json
-from requests_futures.sessions import FuturesSession
-import genet.utils.secrets_vault as secrets_vault
-import genet.utils.spatial as spatial
+
+import genet.outputs_handler.geojson as geojson
 import genet.utils.persistence as persistence
+import genet.utils.secrets_vault as secrets_vault
 import genet.utils.simplification as simplification
-import genet.outputs_handler.geojson as geojson
+import genet.utils.spatial as spatial
+import polyline
+from requests_futures.sessions import FuturesSession
 
 session = FuturesSession(max_workers=2)
 
 
-def send_requests_for_network(n, request_number_threshold: int, output_dir, traffic: bool = False,
+def send_requests_for_network(n, request_number_threshold: int, output_dir,
+                              departure_time: int = -1, traffic_model: str = None,
```

Comment:
I'm not a huge fan of the -1 default as 'now'. 
1. the original default was to `traffic: bool = False` so defaulting the dep time to nothing, not 'now'
2. I think -1 is a bit cryptic, I think defaulting to `None` would be fine, then the departure time is not specified in the request which is the current default, a bit more human readable 
3. for the docs you could specify options: None, or something compatible with google dir specification and link their docs, saves you writing stuff - your method [below](https://github.com/arup-group/genet/pull/101/files#diff-462244c575997d50c1cb3baeec5285e1b7dc38b3df72e8a1430e0a848f6b55ddR78) will simplify too

```
if not None:
  dep time = user input
```